src/pixelator/plot/__init__.py

```diff
@@ -2,15 +2,19 @@
 
 Copyright (c) 2023 Pixelgen Technologies AB.
 """
-
-import matplotlib.pyplot as plt
-import numpy as np
-import scipy
-from matplotlib import cm
-from typing import Literal, Tuple
-
-from pixelator.graph import Graph
-from pixelator.marks import experimental
+import os
+import tempfile
+
+os.environ["MPLCONFIGDIR"] = tempfile.mkdtemp()
```

Comment:
I would rename the dir to something eg. `tempfile.mkdtemp("pixelator-mpl-config")` so it can be reused and does not create a new cache on every invocation.

Reply:
Good suggestion!